centaur/src/it/scala/centaur/reporting/ErrorReporters.scala

```diff
@@ -41,20 +41,20 @@ class ErrorReporters(rootConfig: Config) {
     *
     * @param testEnvironment      The test information, including the name of the test and the attemp.
     * @param ciEnvironment        The information about the CI environment running the test.
-    * @param centaurTestException The exception that occurred while running the test.
+    * @param throwable            The exception that occurred while running the test.
     * @return An IO effect that will log the failure.
     */
-  def logCentaurFailure(testEnvironment: TestEnvironment,
-                        ciEnvironment: CiEnvironment,
-                        centaurTestException: CentaurTestException)
-                       (implicit executionContext: ExecutionContext): IO[Unit] = {
+  def logFailure(testEnvironment: TestEnvironment,
+                 ciEnvironment: CiEnvironment,
+                 throwable: Throwable)
+                (implicit executionContext: ExecutionContext): IO[Unit] = {
     if (errorReporters.isEmpty) {
       // If the there are no reporters, then just "throw" the exception. Do not retry to run the test.
-      IO.raiseError(centaurTestException)
+      IO.raiseError(throwable)
     } else {
-      val listIo = errorReporters.map(_.logCentaurFailure(testEnvironment, ciEnvironment, centaurTestException))
-      AggregatedIo.aggregateExceptions("Errors while reporting centaur failure", listIo).handleErrorWith(throwable => {
-        throwable.addSuppressed(centaurTestException)
+      val listIo = errorReporters.map(_.logFailure(testEnvironment, ciEnvironment, throwable))
+      AggregatedIo.aggregateExceptions("Errors while reporting a failure", listIo).handleErrorWith(throwable => {
+        throwable.addSuppressed(throwable)
```

Comment:
I'm not certain this is wrong, but the shadowing of the formal method parameter `throwable` by this lambda parameter `throwable` makes me unsure...